tools/hyphy/hyphy_meme.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0"?>
+<tool id="hyphy_meme" name="HyPhy-MEME" version="@VERSION@+galaxy0">
+    <description>: Mixed Effects Model of Evolution</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <version_command>HYPHYMP --version | tail -n 1 | awk '{print $1}'</version_command>
+    <command detect_errors="exit_code">
+    <![CDATA[
+    export MPI_NODE_COUNT=\${GALAXY_SLOTS:-1} &&
+    ln -s '$input_file' meme_input.fa &&
+    ln -s '$input_nhx' meme_input.nhx &&
+    echo $gencodeid > tool_params &&
+    echo `pwd`/meme_input.fa >> tool_params &&
+    echo `pwd`/meme_input.nhx >> tool_params &&
+    echo $branches >> tool_params &&
+    echo $p_value >> tool_params &&
+    @HYPHY_INVOCATION@ \$HYPHY_LIB/TemplateBatchFiles/SelectionAnalyses/MEME.bf > '$meme_log'
+    ]]>
+    </command>
+    <inputs>
+        <param name="input_file" type="data" format="fasta" label="Input FASTA file" />
+        <param name="input_nhx" type="data" format="nhx" label="Input newick file" />
+        <param name="gencodeid" type="select" label="Genetic code">
+            <option value="1">Universal code</option>
+            <option value="2">Vertebrate mitochondrial DNA code</option>
+            <option value="3">Yeast mitochondrial DNA code</option>
+            <option value="4">Mold, Protozoan and Coelenterate mt; Mycloplasma/Spiroplasma</option>
+            <option value="5">Invertebrate mitochondrial DNA code</option>
+            <option value="6">Ciliate, Dasycladacean and Hexamita Nuclear code</option>
+            <option value="7">Echinoderm mitochondrial DNA code</option>
+            <option value="8">Euplotid Nuclear code</option>
+            <option value="9">Alternative Yeast Nuclear code</option>
+            <option value="10">Ascidian mitochondrial DNA code</option>
+            <option value="11">Flatworm mitochondrial DNA code</option>
+            <option value="12">Blepharisma Nuclear code</option>
+        </param>
+        <conditional name="rate_cond">
+            <param name="rate" type="select" label="Rate variation">
```

Comment:
It looks like the Rate variation parameter isn't used in this tool. I also didn't see it used (or prompted for) when calling MEME from the command line.